UnitsNet/UnitDefinitions/Density.json

```diff
@@ -122,6 +122,45 @@
           "Abbreviations": [ "slug/ft³" ]
         }
       ]
+    },
+    {
+      "SingularName": "GramPerLiter",
+      "PluralName": "GramsPerLiter",
+      "FromUnitToBaseFunc": "x/1",
+      "FromBaseToUnitFunc": "x*1",
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/L" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "GramPerDeciLiter",
+      "PluralName": "GramsPerDeciLiter",
+      "FromUnitToBaseFunc": "x/1e-1",
+      "FromBaseToUnitFunc": "x*1e-1",
+      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/dL" ]
```

Comment:
Lowercase `l` in `ml`. That is the convention we use for liters in [Volume](https://github.com/anjdreas/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Volume.json).